tests/components/prusalink/test_binary_sensor.py

```diff
@@ -0,0 +1,33 @@
+"""Test Prusalink sensors."""
+
+from unittest.mock import PropertyMock, patch
+
+import pytest
+
+from homeassistant.const import STATE_OFF, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.setup import async_setup_component
+
+
+@pytest.fixture(autouse=True)
+def setup_binary_sensor_platform_only():
+    """Only setup sensor platform."""
+    with (
+        patch("homeassistant.components.prusalink.PLATFORMS", [Platform.BINARY_SENSOR]),
+        patch(
+            "homeassistant.helpers.entity.Entity.entity_registry_enabled_default",
```

Comment:
We have a fixture for this already.

https://github.com/home-assistant/core/blob/ff9937f942828c18c82e8e6ff66ca9f0004fcea8/tests/components/conftest.py#L45-L52